script/get-changes-since

```diff
@@ -1,13 +1,23 @@
 #!/usr/bin/env node --redirect-warnings=/dev/null
 
 const { execFileSync } = require("child_process");
-const { GITHUB_ACCESS_TOKEN } = process.env;
+var { GITHUB_ACCESS_TOKEN } = process.env;
```

Comment:
Should we be using `let` here? I don't think it's ever the case that you want `var` in modern JS:

```suggestion
let { GITHUB_ACCESS_TOKEN } = process.env;
```

Reply:
Lol oops.